sycl/doc/EnvironmentVariables.md

```diff
@@ -69,6 +69,12 @@ Note that all device selectors will throw an exception if the filtered list of d
 
 `(*) Note: Any means this environment variable is effective when set to any non-null value.`
 
+## Controlling DPC++ CUDA Plugin
+
+| Environment variable | Values | Description |
+| -------------------- | ------ | ----------- |
+| `SYCL_PI_CUDA_MAX_LOCAL_MEM_SIZE` | Integer | Specifies the maximum size of a local memory allocation in bytes. If the value exceeds the device's capabilities then `PI_INVALID_BUFFER_SIZE` is thrown in `cuda_piEnqueueKernelLaunch`. |
```

Comment:
I assume that's user-oriented variable and not for debug-only or experimental purpose?
Can you add the default value in description please?

Reply:
The default value is determined by the hardware. For CUDA the default will in most cases be 48KB but can range from 16KB to 163KB.

Reply:
Since I can imagine the range of possible defaults ranging with future versions it should be fine to not mention specific possible default values. That said, it would be good to have it specify that if the variable is unset the value is decided by the device.

Reply:
Yes, in such case we should say it's determined by the device.

Reply:
Change made